docs/src/styles/docs/header.scss

```
@@ -9,6 +9,11 @@
   }
 }
 
+// Remove post-launch
+#main-nav + .docs-header {
+  --docs-dev-center-nav-height: 3rem;
+}
+
```

```suggestion
/* Remove post-launch START */
#main-nav + .docs-header {
  --docs-dev-center-nav-height: 3rem;
}
/* Remove post-launch END */
```

How about making it more clear?